packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -76,18 +81,21 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
   useLocalVideoStartTrigger(!!props.isVideoStreamOn);
   const VideoGalleryMemoized = useMemo(() => {
     return (
-      <VideoGallery
-        {...videoGalleryProps}
-        localVideoViewOptions={localVideoViewOptions}
-        remoteVideoViewOptions={remoteVideoViewOptions}
-        styles={VideoGalleryStyles}
-        layout="floatingLocalVideo"
-        showCameraSwitcherInLocalPreview={props.isMobile}
-        localVideoCameraCycleButtonProps={cameraSwitcherProps}
-        onRenderAvatar={onRenderAvatar}
-      />
+      <>
+        <Announcer announcementString={announcerString} ariaLive={'polite'} />
+        <VideoGallery
+          {...videoGalleryProps}
+          localVideoViewOptions={localVideoViewOptions}
+          remoteVideoViewOptions={remoteVideoViewOptions}
+          styles={VideoGalleryStyles}
+          layout="floatingLocalVideo"
+          showCameraSwitcherInLocalPreview={props.isMobile}
+          localVideoCameraCycleButtonProps={cameraSwitcherProps}
+          onRenderAvatar={onRenderAvatar}
+        />
+      </>
     );
-  }, [videoGalleryProps, props.isMobile, onRenderAvatar, cameraSwitcherProps]);
+  }, [videoGalleryProps, props.isMobile, onRenderAvatar, cameraSwitcherProps, announcerString]);
```

Comment:
+1

We should use `React.memo` instead of `useMemo` when we're trying to "memoize" components.

This PR didn't add the `useMemo`, but it does add more complexity in an already iffy place here.